app/src/main/java/org/schabi/newpipe/util/NavigationHelper.java

```diff
@@ -542,6 +546,43 @@ public static Intent getPlayQueueActivityIntent(final Context context) {
         return intent;
     }
 
+    /*//////////////////////////////////////////////////////////////////////////
+    // Downloading a video without starting a player
+    //////////////////////////////////////////////////////////////////////////*/
+
+    public static void openDownloadDialogWithoutStartingPlayer(
```

Comment:
Please rename this to `openDownloadDialog`. Also, please remove the comment above, as it just repeats the method's name.